### DISCUSSION TITLE
2005/02/03/%E5%8F%88%E6%9C%89%E6%96%B0%E7%9A%84-msn-messager-%E7%89%88%E6%9C%AC%E4%BA%86%EF%BC%8C

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#5] [Announcements]
# 2005/02/03/%E5%8F%88%E6%9C%89%E6%96%B0%E7%9A%84-msn-messager-%E7%89%88%E6%9C%AC%E4%BA%86%EF%BC%8C

不过不是很好用。

https://blog.qizhen.xyz/2005/02/03/%E5%8F%88%E6%9C%89%E6%96%B0%E7%9A%84-msn-messager-%E7%89%88%E6%9C%AC%E4%BA%86%EF%BC%8C

**Comment by ruanqizhen** [comment]
之前博客下的留言：
```
Qizhen
2005年02月24日 09:59
上当了，说是 msn messenger 8.0，其实是个7.0的老beta版。
```
